apps/clock/locales/clock.en-US.properties

```diff
@@ -50,73 +48,46 @@ vibrateOff = Off
-countdown-moreThanADay = The alarm is set for {{days}} {{hours}} from now.
-
-nMinutes={[ plural(n) ]}
-nMinutes[zero]  = less than a minute
-nMinutes[one]   = one minute
-nMinutes[two] = {{n}} minutes
-nMinutes[few] = {{n}} minutes
-nMinutes[many] = {{n}} minutes
-nMinutes[other] = {{n}} minutes
-
-nHours={[ plural(n) ]}
-nHours[zero] =
-nHours[one] = <strong>{{n}} hour</strong>
-nHours[two] = <strong>{{n}} hours</strong>
-nHours[few] = <strong>{{n}} hours</strong>
-nHours[many] = <strong>{{n}} hours</strong>
-nHours[other] = <strong>{{n}} hours</strong>
-
-nAndRemainHours={[ plural(n) ]}
-nAndRemainHours[zero] =
-nAndRemainHours[one] = and <strong>{{n}} hour</strong>
-nAndRemainHours[two] = and <strong>{{n}} hours</strong>
-nAndRemainHours[few] = and <strong>{{n}} hours</strong>
-nAndRemainHours[many] = and <strong>{{n}} hours</strong>
-nAndRemainHours[other] = and <strong>{{n}} hours</strong>
-
-nRemainDays={[ plural(n) ]}
-nRemainDays[zero] =
-nRemainDays[one] = <strong>{{n}} day</strong>
-nRemainDays[two] = <strong>{{n}} days</strong>
-nRemainDays[few] = <strong>{{n}} days</strong>
-nRemainDays[many] = <strong>{{n}} days</strong>
-nRemainDays[other] = <strong>{{n}} days</strong>
-
-nRemainMinutes={[ plural(n) ]}
-nRemainMinutes[zero]  =
-nRemainMinutes[one]   = and <strong>{{n}} minute</strong>
-nRemainMinutes[two] = and <strong>{{n}} minutes</strong>
-nRemainMinutes[few] = and <strong>{{n}} minutes</strong>
-nRemainMinutes[many] = and <strong>{{n}} minutes</strong>
-nRemainMinutes[other] = and <strong>{{n}} minutes</strong>
-
-nSpinnerSeconds={[ plural(n) ]}
-nSpinnerSeconds[zero]  = zero seconds
-nSpinnerSeconds[one]   = one second
-nSpinnerSeconds[two] = {{n}} seconds
-nSpinnerSeconds[few] = {{n}} seconds
-nSpinnerSeconds[many] = {{n}} seconds
-nSpinnerSeconds[other] = {{n}} seconds
-
-nSpinnerMinutes={[ plural(n) ]}
-nSpinnerMinutes[zero]  = zero minutes
-nSpinnerMinutes[one]   = one minute
-nSpinnerMinutes[two] = {{n}} minutes
-nSpinnerMinutes[few] = {{n}} minutes
-nSpinnerMinutes[many] = {{n}} minutes
-nSpinnerMinutes[other] = {{n}} minutes
-
-nSpinnerHours={[ plural(n) ]}
-nSpinnerHours[zero] = zero hours
-nSpinnerHours[one] = {{n}} hour
-nSpinnerHours[two] = {{n}} hours
-nSpinnerHours[few] = {{n}} hours
-nSpinnerHours[many] = {{n}} hours
-nSpinnerHours[other] = {{n}} hours
+countdown_lessThanAnHour = The alarm is set for <strong>{{alarmMinutes}}</strong> from now.
+countdown_moreThanAnHour = The alarm is set for {{alarmHours}} and {{alarmMinutes}} from now.
+countdown_moreThanADay = The alarm is set for {{alarmDays}} and {{alarmHours}} from now.
+
+alarmMinutes={[ plural(minutes) ]}
+alarmMinutes[zero]  = less than a minute
+alarmMinutes[one]   = one minute
+alarmMinutes[two] = {{minutes}} minutes
+alarmMinutes[few] = {{minutes}} minutes
+alarmMinutes[many] = {{minutes}} minutes
+alarmMinutes[other] = {{minutes}} minutes
+
+alarmHours={[ plural(hours) ]}
+alarmHours[zero] =
```

Comment:
Don't leave an empty string, it would only generate confusion in tools. Put a generic plural forms, even if it will remain unused, or 'Zero hours"
```
alarmHours[zero] = <strong>{{hours}} hour</strong>
```